bash_scripts/aprime_atm_diags.bash

```diff
@@ -340,16 +350,643 @@ while [ $j -lt $n_test_cases ]; do
    casename="${case_set[$j]}"
    scratch_dir="${scratch_dir_set[$j]}"
    begin_yr_ts=${begin_yr_ts_set[$j]}
+   end_yr_ts=${end_yr_ts_set[$j]}
 
    ./bash_scripts/plot_time_series.bash $scratch_dir \
 				        $casename \
 				        $begin_yr_ts \
+				        $end_yr_ts \
```

Comment:
For example, I wonder why the changes in aprime_atm_diags up to this point are there in the first place. Seem like things unrelated to this PR.. Do you understand it?

Reply:
Yes, I made these changes to better name log files and check for pre-existing files. I will look at the rebase now. Thanks @milenaveneziani .